### DISCUSSION TITLE
RFC: Gaps in our technical documentation

### DISCUSSION THREAD

**Discussion by JacklynBiggin** [woocommerce/woocommerce#44235] [Request for Comments]
We recently launched a new Woo Developer Documentation site with the aim of improving Woo's developer experience. We consider this a `v1` of the site, and are aware of certain gaps in the documentation. For example:

- There isn't a clear onboarding flow to setup a Woo developer environment and create a basic extension
- Certain categories, such as the Reporting category, are missing introductory information
- Certain documentation - such as WooCommerce Blocks' - isn't available on the docs site yet

**As a developer, what are some gaps that you've run into when trying to use Woo's documentation?** These could be gaps in existing docs, entire categories of documentation that are missing, or anything in between.

If you'd prefer to give feedback privately, you can do so at https://developer.woo.com/docs/feedback.

**Comment by henrywright** [comment]
Hi @JacklynBiggin 

In docs/extension-development/handling-deactivation-and-uninstallation.md I'm not certain the Removing Admin Tasks section is necessary

My understanding of filter hooks is they don't persist. When a plugin is deactivated everything hooked to `woocommerce_get_registered_extended_tasks` will simply go away with no action necessary by the developer
